### ISSUE TITLE
[Feature Request] Backup on Cloud 

### ISSUE THREAD

**Issue by dfrost16** [Jays2Kings/tachiyomiJ2K#1161]
An option for the backup to be auto uploaded to Google Drive or any other cloud drive would be helpful.

**Comment by MindFlayeR57** [comment]
I'll add that it'll be nice to not only backup but also sync between multiple devices that are connected to the same cloud account.

**Comment by dfrost16** [comment]
Indeed, this is the third time I have lost my backup due to formatting my phone. Another nifty feature that I could think off is whilst resorting backup it should also be able to identify and install the extension that would make stuff bit more convenient. Just a suggestion

**Comment by dfrost16** [comment]
I don't think so, and it's not universal and varies on different Android version and Android skin

**Comment by MrGoraj** [comment]
The thread hasn't been touched in almost a year and a half, and the feature would really come in handy, and probably many more people would like it than it seems, since only a few app users post on GitHub.

**Comment by rsmyst** [comment]
I know it's not an easy solution but personally, I set up auto backup into a folder that I have syncthing enabled on. it's a bit convoluted but works perfectly and syncs the backup file to my laptop everytime it's created. like a personal local cloud. 
If you're interested, dig deeper in the below links. 
https://github.com/canton7/SyncTrayzor - Desktop GUI client for syncthing 
https://syncthing.net/ - the actual software (also FOSS). it's CLI based though, so i'd recommend just setting up synctrayzor. 
There's an android app on playstore to go with syncthing too.
